types/office-js-preview/index.d.ts

```diff
@@ -40109,6 +40189,24 @@ declare namespace Excel {
          */
         cellDeleted = "CellDeleted"
     }
+    /**
+     * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+     * @beta
+     */
+    enum RowHiddenChangeType {
+        /**
+         *
+         * Unhidden represents the hidden state of row is Unhidden.
+         *
+         */
+        unhidden = "Unhidden",
+        /**
+         *
+         * Hidden represents the hidden state of row is hidden.
```

Comment:
```suggestion
         * Hidden represents the row is hidden.
```

Reply:
Copy-paste? Seems like this should be:
```suggestion
         * Hidden indicates the row is hidden.
```